README.md

```diff
@@ -1,14 +1,14 @@
 # purescript-record
 
 [![Latest release](http://img.shields.io/bower/v/purescript-record.svg)](https://github.com/purescript/purescript-record/releases)
```

Comment:
Most core libraries have a http://img.shields.io/github/release/purescript/purescript-xxx.svg badge, should we do the same here?